app/assets/stylesheets/signup.css.scss

```diff
@@ -25,6 +25,11 @@ input[type="text"], input[type="alt_email"], input[type="email"], input[type="pa
   display:none;
 }
 
-#sign_up_button:focusout{
-	background-color:grey;
+#sign_up_button{
+  background-color:grey;
+}
+
+#no-js-message{
+  padding-top:20px;
+  color:red;
```

Comment:
A linter would ask for a space after the colon.

Reply:
Fixed.